data/schema/schema_v1.json

```diff
@@ -296,12 +296,19 @@
         },
         "sections": {
           "type": "array",
-          "description": "Title - initial title, title_from_answers - title with information derived from answers",
           "items": {
-            "title": "string",
-            "title_from_answers": "array",
-            "group_order": "array",
-            "description": "Order in which groups will appear during survey",
+            "title": {
+            "type": "string",
+            "description": "Initial title."
+            },
+            "title_from_answers": {
+            "type": "array",
+            "description": "Modified title with information derived from answers."
```

Comment:
I think we should extend this description with "Takes a list of answer ids. Title will be generated from answer values concatenated together with spaces".

Reply:
done